rse/grant-applicants.rst

```diff
@@ -0,0 +1,144 @@
+For grant applicants
+====================
+
+.. warning::
+
+   Grant applicants, please don't rely on this yet.  It will be
+   finalized soon.
+
+   This page is currently (2020-03) a draft for feedback.  Please send
+   corrections to the person that send this page to you, or rse-group
+   at aalto.fi
+
+
+
+If you've decided you would like to use the :doc:`research software
+engineer <index>` services in your project for a long period, you
+might want to write it directly into your grant proposals.  First off,
+your research will be better, second it may make your proposal more
+likely to get funded.
+
+
+
+How it works
+------------
+
+Short-term services are funded by various departments and schools and
+free to projects (part of the "research environment" services).
+Longer term should be funded by projects - either an external grant or
+basic group fund.  There are two ways to write this into a project:
+
+* As a **purchased service**, like usage of different infrastructures.
+  This is flexible, but there is a limit of how much you can include
+  in a grant.  Note that indirect costs and overheads by the research
+  site are not included, so these prices appear higher to
+  cover that.
+
+* As a **research salary**, just like other salaries on your project.
+  This has fewer limits, but is less flexible (you get one person for
+  a designated time period).
+
+Our prices may seem higher than a young researcher assistant, but with
+us you only pay for the actual productive time, and we are multiple
+times more efficient.  We'll also teach your group to work better.
```

Comment:
I would leave this kind of disclaimer out from the introductory part of the document, not to depress the readers already at the beginning.